hack/svc/multi-stream.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import time
+import re
+import getopt
+import sys
+from subprocess import Popen
+
+
+class strims_ffmpeg:
+    def __init__(self, hostAddress="0.0.0.0") -> None:
+        self.hostAddress = hostAddress
+
+    def startStream(self, streamLink: str, streamKey: str) -> str:
+        process = ["ffmpeg"]
+        isUrl = re.match(
+            "^(http[s]?:\/\/(www\.)?|ftp:\/\/(www\.)?|www\.){1}([0-9A-Za-z-\.@:%_\+~#=]+)+((\.[a-zA-Z]{2,3})+)(\/(.)*)?(\?(.)*)?", streamLink)
+        if isUrl:
+            process += ["-reconnect", "1", "-reconnect_at_eof",
+                        "1", "-reconnect_delay_max", "10"]
+
+        process += ["-re", "-y", "-i", streamLink,
+                    "-codec:a", "aac", "-c:v", "copy", "-f", "flv", "-flvflags", "no_duration_filesize",
+                    f"rtmp://{self.hostAddress}:1935/live/{streamKey}"]
+
+        return " ".join(process)
+
+    def main(self) -> int:
+        if shutil.which("ffmpeg") is None and shutil.which("ffprobe") is None:
+            raise Exception("ffmpeg and ffprobe is required")
+        strims = {"default": {"key": "", "link": ""},
+                  "streams": ["https://devstreaming-cdn.apple.com/videos/streaming/examples/img_bipbop_adv_example_fmp4/master.m3u8",
+                              "https://devstreaming-cdn.apple.com/videos/streaming/examples/img_bipbop_adv_example_fmp4/master.m3u8"],  "keys": sys.argv[1:]}

```

Comment:
We can simplify this structure, maybe have a default streams list and we can eventually add a flag to provide additional streams